### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by MobyHrd** [SDClowen/RSBot#643]
well, the bot was working just fine on windows 7, i reinstall a fresh windows, but when i try to open the bot nothing happen, i downloaded again and the setup is completed, but once again i try to open it, absolutely nothing happen, it just like i don't click on it,
OS
windows 7 ultimate 64bit,
netframwork 4.0, 4.8
all the c++ from 2005 to 2022, x64/x86.
directx 9/10/11
flash player
Java
is there something missing ?
thank you

**Comment by SDClowen** [comment]
Detail with photos please.

**Comment by MobyHrd** [comment]
> Detail with photos please.

![1](https://user-images.githubusercontent.com/43768352/236636338-3a221fb1-ef64-4100-ad2b-1be8a51548ef.PNG)

**Comment by MobyHrd** [comment]
> Detail with photos please.

is there something missing ?

**Comment by SDClowen** [comment]
My friend. Do you get any errors when starting RSBot.exe?

**Comment by MobyHrd** [comment]
> My friend. Do you get any errors when starting RSBot.exe?

Nope, how i can get errors when starting ? RSbot.exe won't even start, i clicked on in and nothing happen.

**Comment by SDClowen** [comment]
Try remove .NET SDk and .net 6 runtime desktop and start the RSBot setup exe it will be install the necessary libraries by automatically
